handlers/google_apps_api_v1/readme.md

```diff
@@ -0,0 +1,65 @@
+# Google Apps API V1
+Used for requests to the Google Cloud Apps REST API Client
+
+## Info Values
+**service_account**: An account used to interact with the api.  Instruction on creating a [service account](https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount)
```

Comment:
I got tripped up on this when trying to determine what value needed to be used here when testing. Should indicate here that it is the email address for the account used to interact with the api. It is mentioned in the parameter description so that is good, but I didn't look at that when I was trying to configure values for running through the test harness.